pages/login/index.js

```diff
@@ -30,64 +31,60 @@ export default function Login() {
 
   function updateLogin(e) {
     const { id, value } = e.target;
-    if (id != "checkbox") {
+    if (id !== "checkbox") {
       setLogin((prevValue) => {
         if (id === "email") {
           return { email: value, password: prevValue.password };
         }
+
         return { email: prevValue.email, password: value };
       });
     }
   }
 
   function authenticate(e) {
     e.preventDefault();
-    users.forEach((user) => {
-      if(login.email == user.email) {
-        console.log(true) 
-      }
-    })
+    let successfulLogin = users.find((user) => {
+      return login.email == user.email && login.password == user.password;
+    });
+
+    if (!successfulLogin) {
+      setSignInError(true);
+      return;
+    }
+
+    router.push("/home");
   }
 
   return (
     <div className="login-page">
       <div className="form-container">
-        <form className="container">
+        <form onSubmit={authenticate} type="submit" className="container">
           <i className="fa-solid fa-list-check"></i>
-
           <h1 className="h3 mb-3 fw-normal">Please sign in</h1>
+          {signInError && (
+            <div className="alert alert-danger" role="alert">
+              Incorrect email or password
+            </div>
+          )}
+
           {inputInfo.map((input) => (
-            <LabeledInput
-              id={input.id}
-              key={input.id}
-              divClass={input.divClass}
-              type={input.type}
-              inputClass={input.inputClass}
-              placeholder={input.placeholder}
-              labelClass={input.labelClass}
-              labelFor={input.labelFor}
-              labelText={input.labelText}
-              onChange={updateLogin}
-            />
+            <LabeledInput key={input.id} {...input} onChange={updateLogin} />
           ))}
 
-          {/* <Link href="/home">
-            
-          </Link> */}
-
           <button
-              onClick={authenticate}
-              className="btn btn-dark w-100 py-2"
-              type="submit"
-            >
-              Sign in
-            </button>
+            onSubmit={authenticate}
+            className="btn btn-dark w-100 py-2"
+            type="submit"
+          >
+            Sign in
+          </button>
 
-          <a className="center mt-3" href="">
+          <a className="center" href="">
             Create an account
           </a>
 
-          <p className="center mt-3 text-body-secondary">©2023</p>
+          <p className="center text-dark">©2023</p>
```

Comment:
**issue:** I would recommend using the actual year here. Good practice for you. Try and get the current year. _Remember to use state_